setup-scripts/install.sh

```diff
@@ -81,12 +81,39 @@ msg "Installing node.js@8.11" $WHITE
 nvm install 8.11 --latest-npm
 
 if [ "$(command -v docker)" == "" ]; then
-  msg "Docker is not installed. Please install it before continuing" $RED
+  msg "Docker is not installed." $RED
+  if [[ "$OSTYPE" == "linux-gnu" ]]; then
+    # Get the version of Linux distribution
+    LINUX_DISTRO=$(cat /etc/*-release | grep -E '^ID=' | sed 's/^.*=//' | tr -d \")
+    msg "Looks like you're using Linux." $GREEN
+    msg "To install Docker using the convenience script for Linux, run" $GREEN
+    msg "    $ curl -fsSL https://get.docker.com -o get-docker.sh" $WHITE
+    msg "    $ sudo sh get-docker.sh" $WHITE
+    msg "Alternatively, you can follow Docker's installation steps for Linux (https://docs.docker.com/install/linux/docker-ce/$(echo $LINUX_DISTRO)/)" $GREEN
+  elif [[ "$OSTYPE" == "darwin"* ]]; then
+    msg "Looks like you're using Mac." $GREEN
+    msg "Please use the following link to download and install Docker for Mac Version 2.0.0.3 or greater (this installs Docker Community Edition and Docker Compose)" $GREEN
+    msg "https://docs.docker.com/docker-for-mac/release-notes/#docker-community-edition-2003-2019-02-15" $WHITE
+  else
+    msg "Please install Docker before continuing." $GREEN
+  fi
   exit 1
 fi
 
 if [ "$(command -v docker-compose)" == "" ]; then
```

Comment:
do we need to check the version here? We've had issues with people using versions that didn't do variable interpolation the correct way.

Reply:
Capturing our in-person conversation: we'll check the version of both `docker` and `docker-compose`. If the version is older than ~6 months, we'll fail the install script and tell the user to update. If the version is the current version, we'll warn the user and continue with the installation process.